pkgs/development/python-modules/dbt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchPypi, buildPythonPackage, autopep8 }:
+
+buildPythonPackage rec {
+  pname = "dbt";
+  version = "0.19.0";
+
+  propagatedBuildInputs = [ autopep8 ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "276ced7e9e3cb22e5d7c14748384a5cf5d9002257c0ed50c0e075b68011bb6d0";
+  };
+
+  meta = with lib; {
+    description = ''
+      dbt (data build tool) is a command line tool that helps analysts and
+      engineers transform data in their warehouse more effectively
```

It's a module and CLI tool.

```suggestion
    description = "Python module to transform data";
```

"Extensive" content should go to `longDescription`.